File: benchmark/dbbench2.py
import MySQLdb

conn = MySQLdb.connect(host='localhost', user='user1', passwd='passwd1', db='example1')
sql = 'select * from stocks'
N = 10000
for i in xrange(N):
    #cursor = conn.cursor()                              # 12.3
    cursor = conn.cursor(MySQLdb.cursors.DictCursor)   # 14.1
    cursor.execute(sql)
    L = cursor.fetchall()                              # 10.4 / 12.4
    #L = []
    #row = cursor.fetchone()
    #while row is not None:
    #    #print repr(row)
    #    #L.append(row)                                  # 12.0
    #    row = cursor.fetchone()
    cursor.close()
conn.close()

File: benchmark/dbbench2.php
<?php
$conn = mysql_connect('localhost', 'user1', 'passwd1', 'example1');
mysql_select_db('example1');
$N = 10000;
for ($i = 0; $i < $N; $i++) {
    $result = mysql_query('select * from stocks');
    $list = array();
    while ($row = mysql_fetch_array($result, MYSQL_BOTH)) {
        //print_r($row);
        //var_dump($row);
        $list[] = $row;
    }
    mysql_free_result($result);
}
mysql_close($conn);
?>